crawler-cli/src/main/java/org/gbif/crawler/pipelines/dwca/DwCAToAvroConfiguration.java

```diff
@@ -27,7 +27,15 @@
   @Parameter(names = "--parallelism")
   @NotNull
   @Min(1)
-  public int parallelism;
+  public int poolSize;
+
+  @Parameter(names = "--archive-repository")
+  @NotNull
+  public String archiveRepository;
+
+  @Parameter(names = "--export-avro-base-url")
+  @NotNull
+  public String exportAvroBaseURL;
```

Comment:
exportAvroBaseURL, avroRepository? exportAvroPath? etc..

Reply:
made it as extendedRecordRepository